### PR TITLE
[Fix] Make sure gids are correct before bandwidth measurements

### PR DESCRIPTION
**Pull request by Jackmin801**

### CODE REVIEW

File: src/zeroband/comms.py [hunk 1]
```diff
@@ -159,6 +159,9 @@ def _create_global_pg(self):
         self._logger.debug("Global pg created with %d peers. Timeout of %s", self.global_pg.size(), GLOBAL_PG_TIMEOUT)
 
     def _optimize_ring_ranks(self):
+        self._global_ids = [
+            self.global_store.get(f"gid_{i}").decode("utf-8") for i in range(self.world_info.global_world_size)
```
Comment (samsja) [inline src/zeroband/comms.py:163]:
```suggestion
            self.global_store.get(f"grid_{i}").decode("utf-8") for i in range(self.world_info.global_world_size)
```

typo ?

**Review by samsja — APPROVED** [review]
pre approving

File: src/zeroband/comms.py (hunk 1 above)
Comment (Jackmin801) [reply src/zeroband/comms.py:163]: gid = global id. no typo :p

Comment (samsja) [reply src/zeroband/comms.py:163]: arf I was so proud to have catch a typo

**Pull request merged**